tests/org.jboss.tools.fuse.ui.bot.test/src/org/jboss/tools/fuse/ui/bot/test/RegressionTest.java

```diff
@@ -373,6 +379,51 @@ public void issue_1403() {
 
 	/**
 	 * <p>
+	 * Graphic is disposed when using "Go into"
+	 * </p>
+	 * <b>Link: </b>
+	 * <a href="https://issues.jboss.org/browse/FUSETOOLS-1678">https://issues.jboss.org/browse/FUSETOOLS-1678</a>
+	 */
+	@Test
+	public void issue_1678() {
+
+		ProjectFactory.newProject("test-empty").version(SupportedVersions.CAMEL_LATEST).type(ProjectType.SPRING)
+				.create();
+		CamelEditor editor = new CamelEditor("camel-context.xml");
+		editor.addCamelComponent(new Route(), 100, 100);
+		editor.selectEditPart("Route _route1");
+		FusePropertiesView propertiesView = new FusePropertiesView();
+		propertiesView.activate();
+		propertiesView.selectTab("Details");
+		propertiesView.setProperty(PropertyType.TEXT, "Auto Startup", "true");
+		editor.save();
+		editor.doOperation("Route _route1", "Go Into");
+		assertFalse("Invoke of 'Go Into' failed", editor.isComponentAvailable("Route _route2"));
+		assertFalse("Project 'test-empty' contains 'Graphic is disposed' error", graphicError());
```

Comment:
Yes it's true, i changed that to check Error Log for any error from Fuse Tooling plugins